frontend/packages/dev-console/src/components/import/deploy-image/LoadingDisplay.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+
+import { Loading } from '@console/internal/components/utils';
+import { useFormikContext, FormikValues } from 'formik';
+
+const LoadingDisplay: React.FC = () => {
+  const { values, errors } = useFormikContext<FormikValues>();
+  const isiError = _.get(errors, 'isi.image');
+
+  return (
+    <div>
```

Comment:
Extra div seems unnecessary.

Reply:
I'm not sure why but when I remove it, I get an error. It doesn't like me to start with the next line.

Reply:
`return _.isEmpty(values.isi.image) ? (...JSX...) : null;`